script/latest_snapshot_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# Find the latest snapshot version for the specified branch.
+#
+# Example usage: ./latest_snapshot_version.py 7.x
+
+BRANCH=$1
+
+# The snapshots API has not yet been updated to use "main".
+if [ "$BRANCH" = "main" ]; then
+  BRANCH=master
+fi
+
+curl --silent "https://snapshots.elastic.co/latest/$BRANCH.json" | jq -r .version
```

Comment:
**issue**: curl is suppressing errors. I think we should only ignore stdout and let stderr print the errors (`jq` will fail anyway but at least we would have the error message)

Something like `curl --silent --show-error` should be okay